app/src/main/java/org/simple/clinic/patient/recent/RecentPatient.kt

```diff
@@ -53,41 +53,41 @@ data class RecentPatient(
         ) latestUpdatedAt
```

Comment:
I realised my feedback about not using `*` at all and specifying the fields you require will eliminate the need for the change of using `*` instead of `T.*`

Reply:
@vinaysshenoy Can you please weigh in on this?

Reply:
Can you change the commit message to  -> `Simplify RecentPatient query by removing explicit table names`?